test/powershell/Modules/Microsoft.PowerShell.Utility/NewTemporaryFile.Tests.ps1

```diff
@@ -14,17 +14,18 @@
         A FileInfo object for the temporary file is returned.
 #>
 
-Describe "NewTemporaryFile" -Tags "CI" {
+Describe "New-TemporaryFile" -Tags "CI" {
 
     It "creates a new temporary file" {
-        $tempFile = New-TemporaryFile
-
-        Test-Path $tempFile | Should be $true
-        $tempFile | Should BeOfType System.IO.FileInfo
-
-        if(Test-Path $tempFile)
-        {
-            Remove-Item $tempFile -ErrorAction SilentlyContinue -Force
+        try {
+            $tempFile = New-TemporaryFile
```

Comment:
I think we should remove the `try` block - if the line throw we skip the test silently.
```Powershell
{ $script:tempFile = New-TemporaryFile } | Should -Not -Throw
$script:tempFile| Should Exist
...
```

Reply:
@iSazonov try-finally would still throw and not make the test skip silently due to the missing catch block but I also agree with you that in this special case, we do not need to use the try-finally pattern, so I will remove it

Reply:
Also, the `Should -Not -Throw` syntax is already deprecated in Pester 4.2: https://github.com/pester/Pester/releases/tag/4.2.0

Reply:
I see 4.2 introduce right behavior for `Should -Not -Throw `